terratest/test/k8gb_ingress_annotation_failover_test.go

```diff
@@ -61,6 +61,8 @@ func TestK8gbIngressAnnotationFailover(t *testing.T) {
 	assertGslbStatus(t, options, "test-gslb-annotation-failover", "notfound.cloud.example.com:NotFound roundrobin.cloud.example.com:NotFound unhealthy.cloud.example.com:NotFound")
 	assertGslbSpec(t, options, "test-gslb-annotation-failover", ".spec.strategy.type", "failover")
 	assertGslbSpec(t, options, "test-gslb-annotation-failover", ".spec.strategy.primaryGeoTag", "eu")
+	assertGslbSpec(t, options, "test-gslb-annotation-failover", ".spec.strategy.dnsTtlSeconds", "60")
```

Comment:
aren't values supposed to be ints?

Reply:
@k0da yeah, good point,  but it checks with kubectl based jsonpath behind the scenes, so everything is string in the e2e test context. See https://github.com/AbsaOSS/k8gb/blob/master/terratest/test/helpers.go#L159-L164 for details